examples/VREP/position_and_orientation_control_osc.py

```diff
@@ -0,0 +1,96 @@
+"""
+Running operational space control using VREP. The controller will
+move the end-effector to the target object's position and orientation.
+"""
+import numpy as np
+import pygame
+
+from abr_control.arms import ur5 as arm
+from abr_control.controllers import OSC
+from abr_control.interfaces import VREP
+from abr_control.utils import transformations
+
+
+# initialize our robot config
+robot_config = arm.Config(use_cython=True)
+# create opreational space controller
+ctrlr = OSC(robot_config, kp=200, vmax=10.0)
+
+# create our interface
+interface = VREP(robot_config, dt=.005)
+interface.connect()
+
+# set up lists for tracking data
+ee_track = []
+ee_angles_track = []
+target_track = []
+target_angles_track = []
+
+# control (x, y, beta, gamma) out of [x, y, z, alpha, beta, gamma]
+ctrlr_dof = [True, True, False, False, True, True]
```

Comment:
How can you reach in 3d space if you don't control z? I tried moving the target above the floor manually and it looks like it's reaching towards it, but can't get there because it can't change z. It ends up about 40cm above the target

Reply:
oop yes i'll change the target location in the VREP file. but what's happening here is that we're specifying that we don't care about z, so it'll never match that. what matters here are the x, y, and beta, gamma angles. while the angles matching can't be seen visually they show up in the plot afterwards